clang/docs/ReleaseNotes.rst

```diff
@@ -86,6 +86,9 @@ C++23 Feature Support
 C++2c Feature Support
 ^^^^^^^^^^^^^^^^^^^^^
 
+- Add ``__builtin_is_implicit_lifetime`` intrinsic, which supports
```

Comment:
Why is this called `__builtin_is_implicit_lifetime` instead of `__is_implicit_lifetime` like all the other unary type traits? I thought `__builtin_*` is for functions that return objects

Reply:
CF https://github.com/llvm/llvm-project/issues/98310#issuecomment-2248326726

Reply:
See #98310